### PR TITLE
Adjust view of mobile user actions list

### PR DESCRIPTION
**Pull request by musickiper**
Fix for #5543

### CODE REVIEW

File: bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-list-item/user-list-content/component.jsx [hunk 1]
```diff
@@ -90,9 +91,14 @@ class UserListContent extends Component {
 
   componentDidUpdate() {
     this.checkDropdownDirection();
+    if (this.navbarDefaultClass === null && this.props.navbarRef !== null) {
+      this.navbarDefaultClass = this.props.navbarRef.className;
+    }
   }
 
   onActionsShow() {
+    this.props.navbarRef.className = `${this.navbarDefaultClass}Hide`;
```
Comment (oswaldoacauan) [inline bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-list-item/user-list-content/component.jsx]: You aren't doing this the "React Way". In React, you should ideally not be manipulating anything via the DOM.

Comment (musickiper) [reply bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-list-item/user-list-content/component.jsx]: How about Manipulating virtual dom?

Comment (musickiper) [reply bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-list-item/user-list-content/component.jsx]: I wonder that wrapping navbar by ref and sending it to the children components and manipulate the ref at the children components is wrong way??

**Comment by oswaldoacauan** [comment]
You can fix #5543 via CSS by removing the z-index on the `navbar` and changing `padding-top` to `top` on the user-list and chat. ping me on slack if you need help with that.

File: bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-list-item/user-list-content/component.jsx (hunk 1 above)
Comment (oswaldoacauan) [reply bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-list-item/user-list-content/component.jsx]: you should manipulate any DOM property by yourself, refs are DOMNodes and here you are manipulating the property className without the React knowledge.

**Comment by oswaldoacauan** [comment]
Tested locally, if you open the chat you can't click anything on the navbar (settings dropdown and userlist toogle)

File: bigbluebutton-html5/imports/ui/components/app/component.jsx [hunk 2]
```diff
@@ -67,6 +67,7 @@ class App extends Component {
     this.state = {
       compactUserList: false,
       enableResize: !window.matchMedia(MOBILE_MEDIA).matches,
+      navbarShow: true,
```
Comment (oswaldoacauan) [inline bigbluebutton-html5/imports/ui/components/app/component.jsx]: You can fix it by changing only CSS.

Comment (oswaldoacauan) [reply bigbluebutton-html5/imports/ui/components/app/component.jsx]: Again, if you need help ping me on slack.

**Comment by musickiper** [comment]
I pushed 7462710 for reopening pull request

**Pull request merged**